docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -2649,6 +2649,71 @@ class DetailResource {
 
 WARNING: Unlike with `@Inject` the templates obtained via `RestTemplate` are not validated, i.e. the build does not fail if a template does not exist.
 
+[[vertx_integration]]
+=== Vertx Integration
+
+If you want to use `io.vertx.core.json.JsonObject` as data in your templates, then you will need to add the `quarkus-vertx` extension to your `pom.xml` file (most applications use this extension by default).
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-vertx</artifactId>
+</dependency>
+----
+
+With `io.quarkus.vertx` capability we have a special value resolver for `io.vertx.core.json.JsonObject` which makes it possible to access the properties of a JSON object in a template:
```

Comment:
> With `io.quarkus.vertx` capability

I think this part is a bit too technical. The fact that we use capabilities is internal.
Maybe `With this dependency included, ...`

Reply:
Yes, `With this dependency included,` is better but the "capability" here does not necessarily mean `io.quarkus.deployment.Capability`... I mean, it's ok to use this word with its original meaning.

Reply:
Ah, I thought there's `quarkus-vertx`... ok then and sorry.

Reply:
BTW, talking about capabilities makes sense when in the context of extension development. So if you're targeting this (and given what you said in another comment, it might be the case), then it makes sense to talk about capabilities.

But again, probably missing some context about when you actually need this.

Reply:
And only keep this one (plus the example bellow) with the new wording suggested by Guillaume.